src/env/test/customization.rs

```diff
@@ -112,6 +113,10 @@ impl Customization for TestCustomization {
     fn max_supported_resident_keys(&self) -> usize {
         self.max_supported_resident_keys
     }
+
+    fn slot_count(&self) -> usize {
+        self.slot_count
```

Comment:
There should be a test that this number fits storage constraints.

Reply:
done